src/state/index.ts

```diff
@@ -47,8 +49,11 @@ const store = configureStore({
     multicall,
     lists,
   },
-  middleware: [...getDefaultMiddleware({ thunk: true }), save({ states: PERSISTED_KEYS })],
-  preloadedState: load({ states: PERSISTED_KEYS }),
+  middleware: [
+    ...getDefaultMiddleware({ thunk: true }),
+    save({ states: PERSISTED_KEYS, namespace: `${LS_PREFIX}-${appVersion}` }),
+  ],
+  preloadedState: load({ states: PERSISTED_KEYS, namespace: `${LS_PREFIX}-${appVersion}` }),
```

Comment:
So this will erase user's previous settings? 🤔 
I was thinking of copying available settings to new namespace or something. If it nukes everything then its gonna be annoying for users to find out all their exchange settings, watchlist, etc are gone